src/test/java/mate/academy/validation/EmailValidatorTest.java

```diff
@@ -0,0 +1,52 @@
+package mate.academy.validation;
+
+import javax.validation.ConstraintValidatorContext;
+import org.junit.jupiter.api.Assertions;
```

Comment:
Instead of using `Assertions`, you can use static imports for the methods like `assertTrue` and `assertFalse` to improve readability of the test cases.